Core/Utilities/Grid.py

```diff
@@ -147,7 +147,7 @@ def ldapsearchBDII(filt=None, attr=None, host=None, base=None, selectionString="
 def ldapSite(site, attr=None, host=None):
   """ Site information from bdii.
 
-:param  site: Site as it defined in GOCDB or part of it with globing, for example: \UKI-*
+:param  site: Site as it defined in GOCDB or part of it with globing, for example: \\UKI-*
```

Comment:
```suggestion
:param  site: Site as it defined in GOCDB or part of it with globing, for example: UKI-*
```
IIRC, I commented somewhere that this didn't make any difference? (Neither single nor double slash produce any output in the html documentation)

Reply:
A single slash is a problem as it means something:

![Screenshot 2020-10-21 at 17 33 01](https://user-images.githubusercontent.com/5220533/96742720-82445980-13c3-11eb-85f4-976cecc2e6db.png)

I assumed the slash was put there for a reason without realising it needed to be escaped?

Reply:
From this diff, I seriously doubt that:
https://github.com/DIRACGrid/DIRAC/commit/98a7d8588aa403ae473ba4cfd518352d0ca57c09#diff-f360c4ed4c1d7f46ac67ea9aeb520dad2d4b3af6ba46a93357d883c1ec8c39baR134

It also isn't needed for the input to the ldap search. I could understand if the `*` needed to be escaped, but I don't see any warning in my local compilation runs when just removing all the slashes here.